js/task17.js

```diff
@@ -45,7 +45,8 @@ async function getData() {
   }
 }
 
-function displayImage(res) {
+async function displayImage() {
+  const res = await getData();

```

Comment:
"displayImage" contains "getData".
What's changed?

Here's what I wanted to tell you.
↓
There is only one image.
The assignment text says to make five images.
> 画面遷移してから3秒後に解決されるPromiseが返すオブジェクトを元にimgタグを5つつくる。

With the first five images created, there is no need to get the data again.

Reply:
I'm sorry, I still can't get what you mean. So I don't necessarily write ``const res = await getData();`` in displayImage function?

Reply:
Yes.

Reply:
I just updated displayImage function and so on. Is this what you mean?

Reply:
Sorry, I didn't see your reply an hour ago. Seems like I need more work.

Reply:
No, but it's getting a little closer.

Reply:
I fixed the code. How does it look?